main.go

```diff
@@ -118,22 +129,13 @@ func main() {
 			log.Author = config.Author
 			fmt.Printf("%+v\n", log)
 			os.Exit(0)
+		case "all":
+			beaconLogData := loadLog()
```

Comment:
I didn't think to load the file only on the case of `all` whereas before the file was already loading into disk. This makes more sense to me.